fondat/aws/cloudwatch.py

```diff
@@ -0,0 +1,58 @@
+import aiobotocore
+import fondat.codec
+import fondat.pagination
+import logging
+
+from collections.abc import Iterable
+from fondat.aws import Client
+from fondat.codec import Binary, String
+from fondat.error import InternalServerError, NotFoundError
+from fondat.resource import resource, operation
+from fondat.security import SecurityRequirement
+from typing import Any
+from fondat.monitoring import Measurement, Counter, Gauge, Absolute
+
+
+_logger = logging.getLogger(__name__)
+
+
+def cloudwatch_resource(
+    *,
+    client: Client,
+    security: Iterable[SecurityRequirement] = None,
+):
+
+    if client.service_name != "cloudwatch":
+        raise TypeError("expecting cloudwatch client")
+
+    @resource
+    class Metric:
+        @operation(security=security)
+        async def put_metric(self, measurement: Measurement):
+            await client.put_metric_data(
+                MetricData=[
+                    {
+                        "MetricName": measurement.tags,
```

Comment:
Will this work? `measurement.tags` is a `dict`. I suggest metric name should be `measurement.tags["name"]`.